code/game/verbs/suicide.dm

```diff
@@ -9,6 +9,7 @@
 		return
 	if(confirm == "Yes")
 		suiciding = 1
+		log_game("[key_name(src)] (job: [job ? "[job]" : "None"]) commited suicide at [get_area(src)].")

```

Comment:
Timestamp is already added automatically. I wasn't sure about exact coordinates, it seemed like overkill, but I can add it.